xbmc/settings/dialogs/GUIDialogContentSettings.h

```diff
@@ -76,6 +74,10 @@ class CGUIDialogContentSettings : public CGUIDialogSettingsManualBase
   void InitializeSettings() override;
 
 private:
+  void setLabel2(const std::string &settingid, const std::string &label);
+  void toggleState(const std::string &settingid, bool enabled);
+  void setFocus(const std::string &settingid);
```

Comment:
From the wiki "We use CamelCase for method names and **first letter should be upper case**."
So yep we don't want to mix it.